Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityToolkit.Services.InputSimulation.asmdef

```diff
@@ -1,7 +1,8 @@
 {
     "name": "Microsoft.MixedReality.Toolkit.Services.InputSimulation",
     "references": [
-        "Microsoft.MixedReality.Toolkit"
+        "Microsoft.MixedReality.Toolkit",
+        "Microsoft.MixedReality.Toolkit.SDK"
```

Comment:
I notice we are now adding dependency on .SDK which I think is okay since foundation now has toolkit, services, and SDK. But calling it out in case it is not okay.